### PR TITLE
remove MDC fishtagging from the log4j config

### PR DESCRIPTION
**Pull request by anighanta**
Signed-off-by: anighanta <anirudh.ghanta@hedera.com>

### CODE REVIEW

File: src/main/resources/log4j2.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
 		</RandomAccessFile>
 		<Lambda name="Lambda">
```
Comment (steven-sheehy) [inline src/main/resources/log4j2.xml:34]: Doesn't this Lambda block and any refs to it no longer work without the `aws-lambda-java-log4j2` dependency?

Comment (anighanta) [reply src/main/resources/log4j2.xml:34]: I think we dont need this at all. CloudWatch logs will pick up the STDOUT console output be default. Will test and remove this if possible.

Comment (anighanta) [reply src/main/resources/log4j2.xml]: @steven-sheehy yup, dont need this. Removed this completely

**Review by steven-sheehy — APPROVED** [review]
Tests are failing but diff LGTM

**Pull request merged**